doc/mgr/ansible_orchestrator.rst

```diff
@@ -0,0 +1,45 @@
+
+.. _ansible-orchestrator-module:
+
+====================
+Ansible Orchestrator
```

Comment:
I'd suggest sticking with plain `ansible` as the name.  If we find it becomes necessary to highlight/identify which modules are orchestrator modules, we should do that programmatically rather than with long names.

Reply:
Done!
Now it follows the same pattern that other Orchestratpor modules and it is more elegant in commands. Thanks!